### ISSUE TITLE
Progress?

### ISSUE THREAD

**Issue by Paralaxus** [BuiltBrokenModding/ICBM#270]
Hi.

I've been following this project for a while.
It's been about a year if not more that you guys have been working on this.

Aidan has been neglecting his version of ICBM as well, refusing to update or enhance it or answer.

All I want to know is, will you guys be able to finish this off and make a healthy version anytime soon

I would love to see these module based missiles actually do what they promised to do.

TBH all you need is:
#1. Radar
#2. Electrical Input
#3. ComputerCraft methods
#4. better launch systems.
#5. Ability to traverse worlds.

Everything else seems good. But I think better textures and models would be nice.
I know it seems like I have no idea how much work it takes to do all this, but in all honesty I don't mean to be rude but you guys have had this project for a year. And all i see is a new type of missile being implemented while radars, launch UI's, electrical requirements all remain neglected. I mean we don't need anymore "Cake" or "Dye" missiles. 

We need your modular missiles to work like promised, from launch silos that consume energy and in different tiers so that each one can reach a different difficulty of target.

If you find the time to do some significant overhaul, just keep pushing.
I used to write Lua code for ComputerCraft with the original ICBM Mod and DefenseTech.
I spent around 5-6 hours in a row getting a Anti-Ballistic Missile program working.

Basically what I am trying to say is.
You have too many projects to work on, many of which require too much effort and time.
You are juggling one of the greatest mods of all time with a bunch of new ones you want to make.
If you concentrate all your effort into making a seamless and polished ICBM mod, you will be surprised by the amount of people that will go through your Adf.ly links to download it.

Finish the mod, and reap the benefits of being loved and cash from the links.

Andrew (ICBM Enthusiast since 2013)

**Comment by DarkGuardsman** [comment]
1. Stuff will happen, it takes time especially with me being the only dedicated coder on the team.
2. Most of this stuff is planned but the focus right now is json loaders & 1.8 update
3. Radar will most likely not be added to this mod. There will be a sub mod dedicated just to radar and other support tools. This way people are not forced to install a missile mod to use radar for things like plane detection.
4. I've updated the classic version of ICBM already https://github.com/BuiltBrokenModding/ICBM-Classic it has most of what you want in it already. 
5. Computer craft support is not easy due to dependency injection and stripping. though ICBM does have open computer support for the advanced remote launch system.
6. Its highly likely automated Anti-Ballistics systems will be added. I already have the models ready for the turrets to fire micro missiles to kill small and standard missiles.
